cs/docs/persistence.md

```diff
@@ -127,9 +131,9 @@ Odstraní z databáze knihu s ID 1 a v zápětí vloží novou knihu se stejným
 
 Určitě se podívejte na implementaci metody `persist($entity)` v abstraktní třídě `LeanMapper\Repository`. Mechanismus persistence je z ní dobře patrný.
 
-[Entita](https://codedoc.pub/tharos/leanmapper/v3.1.1/class-LeanMapper.Entity.html) vychází repositáři vstříc svými metodami `isModified()`, `isDetached()`, `detach()`, `getModifiedData()` a `markAsCreated($id, $table, $connection)`. Metoda `isModified()` vrací informaci, zda byla data v entitě od okamžiku jejího vytvoření pozměněná, `isDetached()` vrací informaci, zda se jedná o nově vytvořenou entitu nebo v databázi již existující, `detach()` umožňuje prohlásit entitu za nově vytvořenou, `getModifiedData()` vrací pole pozměněných hodnot (ve formátu položka => pozměněná hodnota) a konečně `markAsCreated($id, $table, $connection)` slouží ke změnu stavu entity z nově vytvořené na již uloženou.
+[Entita](https://codedoc.pub/tharos/leanmapper/v3.1.1/class-LeanMapper.Entity.html) vychází repositáři vstříc svými metodami `isModified()`, `isDetached()`, `detach()`, `getModifiedRowData()`, `attach($id)`, `markAsUpdated()` a `makeAlive($entityFactory, $connection, $mapper)`. Metoda `isModified()` vrací informaci, zda byla data v entitě od okamžiku jejího vytvoření pozměněná, `isDetached()` vrací informaci, zda se jedná o nově vytvořenou entitu nebo v databázi již existující, `detach()` umožňuje prohlásit entitu za nově vytvořenou, `getModifiedRowData()` vrací pole pozměněných hodnot (ve formátu položka => pozměněná hodnota), `attach($id)` slouží ke změnu stavu entity z nově vytvořené na již uloženou, `markAsUpdated()` označí entitu za nepozměněnou a konečně `makeAlive($entityFactory, $connection, $mapper)` poskytuje entitě závislosti, které potřebuje, aby si mohla sama načítat entity, ke kterým má nadefinovanou vazbu.
```

Comment:
Můžu dát odkaz buď na [`master`](https://codedoc.pub/tharos/leanmapper/master/index.html) verzi, nebo [`develop`](https://codedoc.pub/tharos/leanmapper/develop/index.html). Myslím, že master představuje stable verzi, takže bych byl spíš pro ten master. Připravím na to nový PR, který to změní na celém webu.